Added new faculty member Roderich Groß from TU Darmstadt

janrpeters opened the pull request:
Added new faculty member Roderich Groß from TU Darmstadt who is currently still missing.

# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [X] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [X] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [X] Combine multiple updates to a single institution into a **single PR.**

- [X] Only submit one pull request per institution.

- [X] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [X] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [X] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [X] Check to make sure that you have no spaces after commas, or any missing fields.

- [X] Check to make sure the home page is correct.

- [X] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [X] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [X] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [X] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

janrpeters committed:
Added new faculty member Roderich Groß from TU Darmstadt

Added new faculty member Roderich Groß from TU Darmstadt who is currently still missing.

janrpeters commented:
There seems to be a bug in the testing. It yields

exit code 255

which typically signifies an abnormal termination, indicating that the process did not exit as expected.

I assume it has to do with the German Umlaut ß (\ss in latex) in Roderich Groß’ name.

On 27 Oct 2024, at 2:05, github-actions[bot] wrote:

> The validity checks failed. Please look at the logs (click the red X) and correct the errors. Your PR will be closed automatically in 2 days if not fixed.
>
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/emeryberger/CSrankings/pull/7868#issuecomment-2439788752
> You are receiving this because you authored the thread.
>
> Message ID: ***@***.***>

emeryberger commented on csrankings-r.csv:
```diff
@@ -1090,6 +1090,7 @@ Roch Guérin,Washington University in St. Louis,http://www.cse.wustl.edu/~guerin
 Rocky Slavin,University of Texas at San Antonio,http://galadriel.cs.utsa.edu/~rslavin,Y4c96KkAAAAJ
 Roded Sharan,Tel Aviv University,http://www.cs.tau.ac.il/~roded,64G5UgMAAAAJ
 Roderic A. Grupen,University of Massachusetts Amherst,https://www.cics.umass.edu/faculty/directory/grupen_roderic,NOSCHOLARPAGE
+Roderich Groß,TU Darmstadt,https://www.informatik.tu-darmstadt.de/fb20/organisation_fb20/professuren_und_gruppenleitungen/index.en.jsp,s-Unqd8AAAAJ
```
Needs alias entry as well ("Roderich Gross").

janrpeters replied:
How do I create the alias entry? Adding him to dblp-aliases.csv seems not possible as I do not have the privileges!

emeryberger replied:
You need two separate entries, one for each DBLP name.

janrpeters committed:
Added Roderich Groß including his alias Roderich Gross as new faculty member from TU Darmstadt

Added Roderich Groß including his alias Roderich Gross as new faculty member from TU Darmstadt

janrpeters committed:
Corrected the misunderstanding regarding the aliases

janrpeters commented:
I did try it! But it does not work as there is only *ONE* DBLP account

https://dblp.org/pid/g/RoderichGross.html

which has ss in the URL and the German Umlaut ß in his name.

The German umlaut ß seems to be the problem. Even if I write him with ss instead of ß (as done for URLs), I get the same problem as for ß in the name.



On 31 Oct 2024, at 16:17, Emery Berger wrote:

> @emeryberger requested changes on this pull request.
>
>
>
>> @@ -1090,6 +1090,7 @@ Roch Guérin,Washington University in St. Louis,http://www.cse.wustl.edu/~guerin
>  Rocky Slavin,University of Texas at San Antonio,http://galadriel.cs.utsa.edu/~rslavin,Y4c96KkAAAAJ
>  Roded Sharan,Tel Aviv University,http://www.cs.tau.ac.il/~roded,64G5UgMAAAAJ
>  Roderic A. Grupen,University of Massachusetts Amherst,https://www.cics.umass.edu/faculty/directory/grupen_roderic,NOSCHOLARPAGE
> +Roderich Groß,TU Darmstadt,https://www.informatik.tu-darmstadt.de/fb20/organisation_fb20/professuren_und_gruppenleitungen/index.en.jsp,s-Unqd8AAAAJ
>
> You need two separate entries, one for each DBLP name.
>
>
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/emeryberger/CSrankings/pull/7868#pullrequestreview-2408270726
> You are receiving this because you authored the thread.
>
> Message ID: ***@***.***>

emeryberger committed:
Update csrankings-r.csv
